hexrd/ui/hexrd_config.py

```diff
@@ -328,7 +334,11 @@ def load_instrument_config(self, yml_file):
         self.backup_instrument_config()
 
         self.update_visible_material_energies()
-        self.instrument_config_loaded.emit()
+
+        new_detectors = self.get_detector_names()
+        if old_detectors != new_detectors:
+            self.instrument_config_loaded.emit()
```

Comment:
I think that if we are going to use the signal like this, we should probably just change it to call `self.detectors_changed.emit()` instead. We can then probably remove the `instrument_config_loaded` signal from hexrdgui, since it's only emitted here. 

I also see [here](https://github.com/HEXRD/hexrdgui/pull/256/files#diff-73b9467134adf696570c35cf87a32d14R92), `HexrdConfig().detectors_changed` is now connected to `config_changed`, so we can probably then also remove [the line after](https://github.com/HEXRD/hexrdgui/pull/256/files#diff-73b9467134adf696570c35cf87a32d14R93).

Reply:
This makes sense, no need for redundant signals. Fixed to use the `detectors_changed` signal instead.

Reply:
Thanks Brianna!